apps/pipelines/nodes/nodes.py

```diff
@@ -256,7 +262,9 @@ class RouterNode(Passthrough, HistoryMixin):
     model_config = ConfigDict(json_schema_extra=NodeSchema(label="Router"))
 
     prompt: str = Field(
-        default="You are an extremely helpful router", json_schema_extra=UiSchema(widget=Widgets.expandable_text)
+        default="You are an extremely helpful router",
+        min_length=1,
```

Comment:
Why is this length 1?

Reply:
This was my attempt to make it required. This can definitely be improved.